src/pages/blog/[...id].astro

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
Part of the conversion to the new Content Layer API, see https://docs.astro.build/en/guides/upgrade-to/v5/#updating-existing-collections